service/src/main/java/com/commercetools/pspadapter/payone/notification/NotificationProcessorBase.java

```diff
@@ -72,8 +74,14 @@ public final void processTransactionStatusNotification(final Notification notifi
      * @return an immutable list of update actions which will e.g. add an interfaceInteraction to the payment
      * or apply changes to a corresponding transaction in the payment
      */
-    protected abstract ImmutableList<UpdateAction<Payment>> createPaymentUpdates(final Payment payment,
-                                                                                 final Notification notification);
+    protected ImmutableList<UpdateAction<Payment>> createPaymentUpdates(final Payment payment,
+                                                                                 final Notification notification) {
+        final ImmutableList.Builder<UpdateAction<Payment>> listBuilder = ImmutableList.builder();
+        listBuilder.add(createNotificationAddAction(notification));
+        listBuilder.add(setStatusInterfaceCode(notification));
+        listBuilder.add(setStatusInterfaceText(notification));
```

Comment:
Is [`PaymentStatus#state`](http://dev.commercetools.com/http-api-projects-payments.html#paymentstatus) (not transaction state) set somewhere? Looks like it is missed from the payment transaction response.